### ISSUE TITLE
No module named 'odoo'

### ISSUE THREAD

**Issue by flonle** [camptocamp/pytest-odoo#56]
I don't understand, the script is invoked by pytest, under Python site-packages. How could it ever find Odoo there to import?

**Comment by simahawk** [comment]
it must be installed in the same python env where odoo is installed.
And you have to follow this instructions https://github.com/camptocamp/pytest-odoo#usage

**Comment by flonle** [comment]
My question was not how to get it to work, I can do that just fine. Let me rephrase: I don't see how anyone's Odoo directory can end up in pythonpath automatically, so either I'm missing something or the readme is.

**Comment by Jerther** [comment]
@MeViMo  I'm using PyCharm for development, and that's how I do it:
![image](https://user-images.githubusercontent.com/5856745/221930085-a7e000e2-9780-4b1e-9774-89cc99f8d292.png)

So I just have Odoo in my project and use PyCharm's "Project Dependencies".

**Comment by sebalix** [comment]
@MeViMo you have to install `odoo` as a Python package in the same Python environment where you installed `pytest`, let's say you git-cloned Odoo in `./src/`:
```sh
$ cd ./src/
$ pip install -r requirements.txt
$ pip install -e .
$ python
>>> import odoo
>>> odoo
<module 'odoo' from '[...]/src/odoo/__init__.py'>
>>>
$ which odoo
[...]/src/.venv/bin/odoo
```

**Comment by flonle** [comment]
Damn, my bad, I didn't even realize Odoo had a setup.py and people were installing it as a Python package.

**Comment by sebalix** [comment]
I checked on the official documentation and this install is not documented (or partially, some bits are missing)...
